app/emails/templates/seller_edit_notify_auth_rep.md

```diff
@@ -0,0 +1,14 @@
+Hi {{ auth_rep_name }},  
+  
+You have been nominated as the authorised representative of {{ supplier_name }} on the Digital Marketplace. From {{ ma_start_date }}, you must accept the new Master Agreement before you can apply for opportunities.  
+  
+You must [review the Master Agreement]({{ frontend_url }}/2/seller-edit/896/representative) on behalf of your business before you can proceed.  
```

Comment:
Looks like that `896` needs to be replaced with a `{{ supplier_code }}`.

Reply:
fixed